install.sh

```diff
@@ -229,16 +230,16 @@ function download_check () {
     cd "${CREW_BREW_DIR}"
 
     #download
-    echo "Downloading ${1}..."
+    echo -e "${BLUE}Downloading ${1}...${RESET}"
```

Comment:
There is already an alias to `echo -e` on line 25.  IOW, no need to add the `-e` option.

Reply:
The `echo` alias doesn't work on my laptop :(